### ISSUE TITLE
Communicator: Use `close` instead of deprecated `stop`

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-core#4720]
This is from the [warnings of the test suite](https://github.com/aiidateam/aiida-core/pull/4719/checks?check_run_id=1858668311#step:9:558)

```
  /home/runner/work/aiida-core/aiida-core/aiida/cmdline/commands/cmd_status.py:116: UnsupportedWarning: stop is unsupported as of 0.7.0. Use close() instead.  Unlike stop(), close() is permanent and the class should not be used again after this.
    comm.stop()
```

@sphuber Could you check whether these calls can simply be replaced or whether you are using them in a way that one expects it to be possible to reopen the communicator again?

**Comment by sphuber** [comment]
In this case it is fine to call `close` because after that the communicator is no longer used as the `verdi` command will terminate.

**Comment by ltalirz** [comment]
Thanks! I've added the fix to #4719
